astroid/brain/brain_namedtuple_enum.py

```diff
@@ -113,6 +115,17 @@ def infer_named_tuple(node, context=None):
     """Specific inference function for namedtuple Call node"""
     class_node, name, attributes = infer_func_form(node, nodes.Tuple._proxied,
                                                    context=context)
+
+    call_site = arguments.CallSite.from_call(node)
+    try:
+        func = next(extract_node('import collections; collections.namedtuple').infer())
+        rename = next(call_site.infer_argument(func, 'rename', context)).bool_value()
+    except (InferenceError, StopIteration):
```

Comment:
If it raises StopIteration, then it is a bug. Let's not mask it, so we could know what exactly fails.